### ISSUE TITLE
Client throws TypeError, doesn't load on test chain (networkid 10101)

### ISSUE THREAD

**Issue by tinybike** [AugurProject/augur#146]
Client doesn't work on the test chain (networkid 10101).  I get this error in my Firebug console:
```
TypeError: this.contractAddress[this.networkId] is undefined
http://localhost:8080/app.js
Line 46740
```
This error goes away if I check out commit 5869ee47e6fed09500cf793e48a1df4304df87b7, so it was introduced in the last couple commits.

**Comment by scottzer0** [comment]
i think this is fixed now.  can you confirm?  i'm using augur-contracts, so it should be aware of any network id in there.

**Comment by tinybike** [comment]
It's throwing a different error now:
```
TypeError: result.toNumber is not a function
http://localhost:8080/app.js
Line 44870
```
(Not sure if the original error is fixed or not -- it may be "underneath" this error...)

**Comment by tinybike** [comment]
Fixed in 3c5a5a0adc4d73a857c17a1d2c7ec2c66cd528b9.
